Jenkinsfile.deploy

```diff
@@ -89,7 +76,25 @@ node('multiarch-' + env.BASHBREW_ARCH) { ansiColor('xterm') {
 			sh """#!/usr/bin/env bash
 				set -Eeuo pipefail
 
-				${ shell }
+				commands=( ${ shell } )
+				for c in "${commands[@]}"; do
+					tries=3
+					while ! output="$( { $c; } |& tee /dev/stderr )"; then
+						# check to see if we hit the docker hub rate limit
+						if grep -q 'TOOMANYREQUESTS' <<<"$output"; then
```

Comment:
Is `TOOMANYREQUESTS` something `crane` outputs, or something that just happens to be in the error text from Docker Hub right now (and thus might change in the future)?

I don't think this is worth doing much about (ie, this is fine as-is because the failure mode would be that we just ... don't retry and thus the problem becomes obvious), but something to consider in the future.